src/lib/draw/Toolbox.svelte

```diff
@@ -55,17 +55,19 @@
     "street-view": new EventHandler(),
   };
 
-  MapEvents.forEach((eventName) => {
+  for (let eventName of MapEvents) {
```

Comment:
Out of interest why do we prefer `for` over `forEach`?

Reply:
Ah, will revert. I was trying a bunch of keyof/indexable type things that didn't work out. We use regular for loops in most places in the code, so it's default muscle memory for me

Reply:
Discussed offline - in this case, not likely to share a separate function body anywhere. We'll prefer forEach / separate functions + functional methods especially in cases like #278 with the filter. Will go fix that up now